clone_repos.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash -aux
+MIRROR="https://github.com"
+PYTHON="python"
+
+function install_rpm () {
+    yum install -y $@
+}
+
+function install_git () {
+    git clone $MIRROR/$2 -b $3 $1
+    cd $1
+    $PYTHON setup.py install
+    cd ..
+}
+
+# install some base RPM packages
+install_rpm python-setuptools python-devel python-pbr openssl-devel libxml2-devel libxslt-devel gpm-devel
+#TODO most of these we can uninstall after installation...
```

Comment:
Uninstalling anything is an example of things that cannot be reliably done if you mix RPM and source python packages.